libkernelflinger/Android.mk

```diff
@@ -262,7 +262,8 @@ else
     LOCAL_CFLAGS += -DPREDEF_DISK_BUS=\"$(BOARD_DISK_BUS)\"
 endif
 
-LOCAL_C_INCLUDES := $(LOCAL_PATH)/../include/libkernelflinger \
+LOCAL_C_INCLUDES := $(KERNELFLINGER_LOCAL_PATH)/include \
+		$(LOCAL_PATH)/../include/libkernelflinger \
```

Comment:
since "../" is  cleaned not only from C code,  but also Makefile  (Android.mk  line 64),  will other "../" be cleaned from Makefile? 
for example,  change 
$(LOCAL_PATH)/../include/libkernelflinger   
to
$(KERNELFLINGER_LOCAL_PATH)/include/libkernelflinger

Reply:
Yes yes. Like I said in emails. This pull request removes subfolders from include-directory. Next pull (fortcoming) request removes all  ".." from all C-, H- and MK-files. Also next pull request modifies all "#include"-lines to our coding style agreement.